### PR TITLE
Support for musical sources (and prototype annotations)

### PR DESCRIPTION
**Pull request by haogatyp**

### CODE REVIEW

File: Resources/Private/Less/modules/score.less [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/*
+ *
+ * Score
+ * ================================================
+ * Specials for the Score view
+ *
+ * Author: Thomas Jung <thomas@tjwd.de>
+ * License: All rights reserved
```
Comment (stweil) [inline Resources/Private/Less/modules/score.less:8]: This license is not compatible with GPL-3.0, so the code cannot be used for the DFG viewer.

**Review by stweil — CHANGES_REQUESTED** [review]

**Comment by sebastian-meyer** [comment]
@haogatyp Please resolve the merge conflicts.

File: Resources/Private/Less/modules/score.less (hunk 1 above)
Comment (stweil) [reply Resources/Private/Less/modules/score.less:8]: See for example this discussion about "All rights reserved" in the context of the GPL:
https://lists.debian.org/debian-legal/2007/06/msg00249.html.

Comment (sebastian-meyer) [reply Resources/Private/Less/modules/score.less:8]: You are right!

@gnuj Can we do something about this? May I put all the files under GPL license?

Comment (gnuj) [reply Resources/Private/Less/modules/score.less:8]: @sebastian-meyer Yes, of course! No problem on my side if the licence lines are removed. These are very old rudiments, and I think the same applies to the EOF statements in the last line. I would be happy to look into this in a separate branch in the course of the coming week. Thats okay? 🤔

Comment (stweil) [reply Resources/Private/Less/modules/score.less:8]: Good idea.

**Comment by stweil** [comment]
> There is again a merge conflict to be solved.

In addition there are several issues reported by Codacy which I think could be fixed.

File: Resources/Private/JavaScript/dfgviewerScripts.js [hunk 2]
```diff
@@ -138,20 +175,28 @@ $(document).ready(function() {
     } else {
         $('.fwds, .backs')
             .on('mouseenter', function () {
-                $(this).addClass('over');
+                $(this).addClass('over').siblings('[class$=' + $(this).attr('class').split(' ')[0].slice(1) + ']').addClass('over');
+
+
```
Comment (sebastian-meyer) [inline Resources/Private/JavaScript/dfgviewerScripts.js:180]:
```suggestion
```

File: Resources/Private/Less/all.less [hunk 3]
```diff
@@ -17,11 +17,14 @@
...
 
 // Basic structures
 @import "structure.less";
 
 // Modules
 @import "modules/fulltext.less";
+@import "modules/score.less";
 @import "modules/gridview.less";
 @import "modules/sidebar.less";
+@import "modules/home.less";
```
Comment (sebastian-meyer) [inline Resources/Private/Less/all.less:30]:
```suggestion
@import "modules/home.less";

```

File: Resources/Private/Plugins/Kitodo/Templates/PageView/Main.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <f:comment>
-<!--
+    <!--
```
Comment (sebastian-meyer) [inline Resources/Private/Plugins/Kitodo/Templates/PageView/Main.html:2]:
```suggestion
<!--
```

**Review by sebastian-meyer — CHANGES_REQUESTED** [review]

File: Resources/Private/JavaScript/dfgviewerScripts.js [hunk 5]
```diff
@@ -19,25 +19,62 @@ $(document).ready(function() {
...
     });
 
+    // section toggle inside the sidebar on larger screens
+    $('.control-bar .control-bar-container h3').on('click', function () {
+        $(this).parent().toggleClass('section-hidden');
+    });
+
+    // toggle for full metadata display in sidebar
+    if ($('.control-bar .metadata-basic dl.tx-dlf-metadata-titledata').length > 1) {
+        metadataToggleLabelMore = ($('html[lang^="de"]')[0]) ? 'mehr Metadaten anzeigen' : 'more Metadata';
```
Comment (sebastian-meyer) [inline Resources/Private/JavaScript/dfgviewerScripts.js:33]:
```suggestion
        var metadataToggleLabelMore = ($('html[lang^="de"]')[0]) ? 'mehr Metadaten anzeigen' : 'more Metadata';
```

File: Resources/Private/JavaScript/dfgviewerScripts.js [hunk 6]
```diff
@@ -19,25 +19,62 @@ $(document).ready(function() {
...
+        metadataToggleLabelLess = ($('html[lang^="de"]')[0]) ? 'weniger Metadaten anzeigen' : 'less Metadata';
```
Comment (sebastian-meyer) [inline Resources/Private/JavaScript/dfgviewerScripts.js:34]:
```suggestion
        var metadataToggleLabelLess = ($('html[lang^="de"]')[0]) ? 'weniger Metadaten anzeigen' : 'less Metadata';
```

**Review by sebastian-meyer — CHANGES_REQUESTED** [review]
Last fixes for the remaining static code analyzer issues. Also, please have a look at the Codacy issues: https://app.codacy.com/gh/slub/dfg-viewer/pull-requests/230/issues

**Review by sebastian-meyer — APPROVED** [review]
Good work! This is ready for testing, now!

The only remaining issue is with the new Grunt version. Could you please check that?
(see https://github.com/slub/dfg-viewer/actions/runs/9877302151/job/27494458932?pr=230)

**Comment by chrizzor** [comment]
It looks like Thomas Jung @gnuj has set the version of grunt in the package.json to ^1.6. The installed version therefore does not match the specification in the package-lock.json file. Perhaps Thomas Jung still needs to commit the lock file? I'm not sure.

**Comment by sebastian-meyer** [comment]
@beatrycze-volk From my point of view, this is ready for merging. The corresponding PR https://github.com/kitodo/kitodo-presentation/pull/1281 has been merged.

This can be tested at https://test.dfg-viewer.de

The remaining Grunt issue should be easily fixed by running `npm update` once. Unfortunately I can't do that because I don't have access to the source branch.

**Comment by gnuj** [comment]
> The remaining Grunt issue should be easily fixed by running `npm update` once. Unfortunately I can't do that because I don't have access to the source branch.

Thats completely correct. This will probably be significantly less work than if I start a complete new branch including a merge request. If this is important for organisational reasons just hit me up.

**Comment by sebastian-meyer** [comment]
Since we all can't access the source branch (and @chrizzor isn't available for the next two months), I propose merging this, then immediately run `npm update` and push the changes again. Is that OK, @beatrycze-volk?

**Comment by beatrycze-volk** [comment]
Hi @sebastian-meyer, I currently have no access to my computer so please feel free to make it :)

**Comment by sebastian-meyer** [comment]
OK, you are the release manager, I'll do as you say! ;o)

**Comment by stweil** [comment]
I'm afraid that Resources/Public/Css/allStyles.css.map should not have been added. Pull request #294 removes it again.

File: Resources/Private/Plugins/Kitodo/Templates/PageView/Main.html [hunk 7]
```diff
@@ -9,12 +9,156 @@
...
+        </div>
+    </f:then>
+    <f:else>
+        <div id="tx-dfgviewer-map-0" class="tx-dlf-map"></div>
+        <div id="svg_panel">
+            <div id="player"></div>
+            <div id="svg_output" style="overflow:hidden;"></div>
+        </div>
+
+        <f:comment>
```
Comment (beatrycze-volk) [inline Resources/Private/Plugins/Kitodo/Templates/PageView/Main.html:153]: As previously mentioned no commented code.

File: Resources/Private/Templates/Kitodo.html [hunk 8]
```diff
@@ -3,7 +3,7 @@
 <div xmlns="http://www.w3.org/1999/xhtml" lang="en" xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers">
 
 	<f:section name="Content">
-		<f:format.raw>{content}</f:format.raw>
+		<f:format.raw>{ccontent}</f:format.raw>
```
Comment (beatrycze-volk) [inline Resources/Private/Templates/Kitodo.html:6]: Why the name is changed?

File: Resources/Public/JavaScript/OLStyles.js [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    Mainly new styles for hover and select
+    interactions with the canvas object.
+
+
+!*/
+/*global ol, dlfViewerOLStyles*/
+/**
+ * @return {ol.style.Style}
+ */
+dlfViewerOLStyles.hoverStyle = function () {
```
Comment (beatrycze-volk) [inline Resources/Public/JavaScript/OLStyles.js:14]: Are open layer not included in kitodo presentation? Why additionally here?

**Review by beatrycze-volk — COMMENTED** [review]
@sebastian-meyer Unfortunately this PR has broken few functionalities inside the DFG Viewer. eg. the calendar view for the newspapers is gone, the fulltext window is gone. I have to revert those changes.

I have left few review comments, but they contain only the problems discovered on the first look. After they are fixed and the removed functionalities are restored, I will make more through review.

**Comment by sebastian-meyer** [comment]
@chrizzor has to take a look at this and fix these issues, since it's his work. But he won't be back before end of september.

**Pull request merged**